src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/service/RemittalDocumentsAppender.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.service;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.RemittalDocument;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.field.IdValue;
+
+@Service
+public class RemittalDocumentsAppender {
```

Comment:
What is the difference between this and Appender<T>?

Reply:
DocumentsAppender appends Documents & this class 'prepends' the RemittalDocument objects to an already existing collection.